.github/workflows/create_archive_and_notes.sh

```diff
@@ -31,14 +31,20 @@ Add to your \`MODULE.bazel\` file:
 
 \`\`\`starlark
 bazel_dep(name = "rules_python", version = "${TAG}")
+python = use_extension("@rules_python//python:extensions.bzl", "python")
 
-pip = use_extension("@rules_python//python:extensions.bzl", "pip")
+python.toolchain(
+    python_version = "3.10",
+)
+use_repo(python, "python_3_10", "python_aliases")
```

Comment:
I'd rather omit the toolchain part and python_aliases part and put those into a dedicated doc about using bzlmod. The reason is if and how you should do it depends on whether you're the root module or a submodule (which is further complicated by every submodule is the root module for itself during development).

Reply:
I'd be fine with parring this down to just the `bazel_dep()` call. That's about the minimum you need, and it contains the release-specific information (the version).

Reply:
I am on the fence.  Do many people use the bare minimum?